### PR TITLE
Add test for sample data + update it.

### PR DESCRIPTION
**Pull request by bhearsum**
@rail - this is a quick fix for the sample data. I still don't think we should be packaging it with the library...it's too big.

### CODE REVIEW

File: auslib/test/test_db.py [hunk 1]
```diff
@@ -546,6 +547,24 @@ def testMultiplePrimaryHistoryDeleteRollback(self):
             self.assertEquals(len(ret), 5, msg=ret)
 
 
+class TestSampleData(unittest.TestCase, MemoryDatabaseMixin):
+    """Tests to ensure that the current sample data (used by Docker) is
+    compatible with the current schema."""
+    sample_data = path.join(__file__, "..", "..", "scripts", "sample-data.sql")
+    sample_data = "/home/bhearsum/repos/master/balrog/scripts/sample-data.sql"
```
Comment (rail) [inline auslib/test/test_db.py]: no such file or directory :P

Comment (bhearsum) [inline auslib/test/test_db.py]: Oops...I thought I took that out! No wonder it was passing :D. I'll fix this.

Comment (rail) [inline auslib/test/test_db.py]: otherwise lgtm

**Comment by rail** [comment]
:shipit:

**Pull request merged**